docs/build/references/command-line-interface.md

```diff
@@ -126,15 +138,23 @@ Each node runs an HTTP server that provides the APIs for interacting with the no
 
 `--http-tls-cert-file` (string, file path):
 
-This argument specifies the location of the TLS certificate used by the node for the HTTPS server. This must be specified when `--http-tls-enabled=true`. There is no default value.
+This argument specifies the location of the TLS certificate used by the node for the HTTPS server. This must be specified when `--http-tls-enabled=true`. There is no default value. This flag is ignored if `--http-tls-cert-file-content` is specified.
+
+`--http-tls-cert-file-content` (string):
+
+As an alternative to `--http-tls-cert-file`, it allows specifying base64 encoded content of the TLS certificate used by the node for the HTTPS server. This must be specified when `--http-tls-enabled=true`.
```

Comment:
Does this take into account that TLS cert and key file is already base64 encoded? Are the users supposed to base64 encode the contents again (with the leading and trailing header), or just use the base64 contents of the file? Might be good to be specific.

Reply:
Same question applies to other TLS flags as well.

Reply:
Yes the full file content must be base64 encoded and passed as flag argument. Remarked this in the docs, so hopefully it's clearer